Vestibulum.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title></title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href="css/style.css" rel="stylesheet">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div>
+            <header class="header"></header>
+            <div class="banner"></div>
+            <div class="content">
+                <div class="list-line">
+                    <a class="line" href=""></a>
+                    <a class="line" href=""></a>
+                    <a class="line" href=""></a>
+                </div>
+                <div class="recommend-topservice">
+                    <div>
+                        <h5 class="topservice-title color-gray set-parent-background">Some of our top services
+                            <p class="topservice-describe color-gray set-parent-background">Ut eleifend libero sed</p>
+                        </h5>                        
+                    </div>
+                    <a class="view-more background-red color-white">VIEW MORE</a>
+                </div>
+                <div class="option-list">
+                    <div class="option-item">
+                        <div class="item-text background-white">
+                            <img class="icon set-parent-background" src="./ic_suspendisse.png" alt="">
+                            <h6 class="item-title color-gray set-parent-background">SUSPENDISE
+                                <p class="item-description color-gray set-parent-background">Quisque id tellus svsf dggfd sdf sd dsf sd  dsfd sd fds safsd asd  sdf s fs df s   sdf s f sdf  fs ffs fsfs f e w</p>
+                            <h6>                            
+                        </div>
+                        <a class="read-more-text background-red color-white" href="">read more</a>                        
+                    </div>
+                    <div class="option-item">
+                        <div class="item-text background-white-f8">
+                            <img class="icon set-parent-background" src="./ic_maecenas.png" alt="">
+                            <h6 class="item-title color-gray set-parent-background">MAECENAS
+                                <p class="item-description color-gray set-parent-background">Quisque id tellus svsf dggfd sdf sd dsf sd  dsfd sd fds safsd asd  sdf s fs df s   sdf s f sdf  fs ffs fsfs f e w</p>
+                            <h6>                            
+                        </div>
+                        <a class="read-more-text background-blue color-white" href="">read more</a>                        
+                    </div>
+                    <div class="option-item">
+                        <div class="item-text background-white-f8">
```

Comment:
this `<div>` contains `img`, `text`, not only text, so name shouldn't `item-text`, u can rename `item-wrapper`